### PR TITLE
Add adjecent styles for headings after lists

### PR DESCRIPTION
**Pull request by dashouse**
This adds padding to the top of headings that follow a list

### CODE REVIEW

**Comment by 36degrees** [comment]
Worth ensuring there's an example for this on the typography example page?

**Comment by dashouse** [comment]
@36degrees Updated

File: src/globals/scss/core/_typography.scss [hunk 1]
```diff
@@ -139,14 +139,17 @@
   }
 
   .govuk-body-m  + .govuk-heading-l,
-  .govuk-body-s  + .govuk-heading-l {
+  .govuk-body-s  + .govuk-heading-l,
```
Comment (hannalaakso) [inline src/globals/scss/core/_typography.scss]: Does this rule somehow relate to lists?

File: src/globals/scss/core/_typography.scss [hunk 2]
```diff
@@ -139,14 +139,17 @@
...
+  .govuk-list + .govuk-heading-l {
     @include govuk-responsive-padding($govuk-spacing-responsive-4, "top");
   }
 
   .govuk-body-m + .govuk-heading-m,
   .govuk-body-s + .govuk-heading-m,
+  .govuk-list + .govuk-heading-m,
   .govuk-body-m + .govuk-heading-s,
-  .govuk-body-s + .govuk-heading-s {
+  .govuk-body-s + .govuk-heading-s,
```
Comment (hannalaakso) [inline src/globals/scss/core/_typography.scss]: Similar here, does this relate to lists styling?

File: src/globals/scss/core/_typography.scss (hunk 1 above)
Comment (36degrees) [reply src/globals/scss/core/_typography.scss]: The only thing that's changed is the opening parenthesis has been changed to a comma to allow the extra selectors to be added to the list…

Comment (hannalaakso) [reply src/globals/scss/core/_typography.scss]: So it has, I must be tired 😅

**Pull request merged**